index.js

```diff
@@ -1,184 +1,249 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
-
-
+// console.log("%cQuestion 1", "color: red")
+ 
+let counter = 1
+counter = 2
+console.log(counter)
+// console.log("%c----------", "color: red")
+ 
+ 
+ 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
+// *** after testing, comment the line causing the error back in
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
-
-
-
+// console.log("%cQuestion 2", "color: red")
+ 
+const name = "Sam"
+ 
+ 
+ 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
-
-
-
+// console.log("%cQuestion 3", "color: red")
+ 
+function drinkWater(current_thirst_level){
+ console.log("Man, I sure am thirsty.")
+ current_thirst_level -=1
+ console.log("Ahh that hits the spot.")
+ return current_thirst_level
+}
+ 
+console.log(drinkWater(12))
+ 
+ 
+ 
+ 
```

Comment:
Great work. Remember that snake case is the standard or norm for defining ruby variables but camel case is the preferred way in JS. 

ruby = current_thirst_level 
JS = currentThirstLevel

Bonus: How would you write the above function as an arrow function?